src/main/archetype/ui.tests/test-module/lib/commons.js

```diff
@@ -14,7 +14,7 @@
  *  limitations under the License.
  */
 const conf   = require('./config');
-const moment = require('moment');
+const format = require('date-fns').format;
```

Comment:
what was the reason to replace `moment` with `format`? Is `moment` somehow vulnerable?